ckanext/saml2auth/views/saml2auth.py

```diff
@@ -130,9 +130,8 @@ def disable_default_login_register():
 
 
 saml2auth.add_url_rule(u'/acs', view_func=acs, methods=[u'GET', u'POST'])
-saml2auth.add_url_rule(u'/user/saml2login', view_func=saml2login)
```

Comment:
I don't think you want to remove this, in case folks are using `/user/saml2login` for saml, and `/user/login` for native login.

Reply:
ah, I didn't see your last commit.